### ISSUE TITLE
.env 

### ISSUE THREAD

**Issue by Samaamro20** [FACK1/FoodRecipes#44]
- should not uplode .env file with your project

**Comment by shahenazmonia** [comment]
Right, also why you have two files contain API key? .env file and .env.example file
Just you need one file to ignore sensitive data

**Comment by mhmdtshref** [comment]
We ignored it, but it still into the master, will remove it.
